feedbag/day2/1.passwords.py

```diff
@@ -0,0 +1,1024 @@
+"4-10 p: pkppshsfmp",
+"10-11 g: gmgggzvzmgjgggzqxggd",
+"2-4 j: fjvjjqf",
+"5-19 h: cstkhhrctkxhhsfzpph",
+"6-16 s: sssssssssssssssks",
+"17-18 v: vvvvvvvvsvvvvvvvtv",
+"2-5 j: zjjjjj",
+"2-3 r: xrrrqlxl",
+"17-19 c: cccrcccgcwcchccccdcc",
+"7-14 l: lllllllllllllcsl",
+"12-14 p: jlmhpjgdzjkpnbmnbk",
+"17-18 n: nnnnnnnnnnnnnnnnqv",
+"8-10 b: lqzrtbzbrb",
+"8-11 x: xgxxfxxxxkxx",
+"5-9 x: xxxsnxnxk",
+"2-11 t: stxbxwssmgthttx",
+"6-13 t: ktbktrtqcnlrgtvtwt",
+"10-12 q: qqqqqqqqqrqq",
+"14-15 q: znptzmgllrfrqrf",
+"2-13 f: ffffffcffffff",
+"9-10 h: hrbhnhhhhhhhh",
+"8-14 k: lskskkkfzkkkkmkkm",
+"1-3 g: kbmswhmghvwvnwxzzk",
+"2-4 v: vttv",
+"2-3 h: qhhmhghbh",
+"6-13 t: ttgtttttktttdttt",
+"7-9 v: vvrrvvvvvv",
+"5-6 z: hcfwdr",
+"5-13 g: sgmgggcggggggjrggggk",
+"10-14 k: kzkkkkkkdskkktkkk",
+"1-2 j: ptjj",
+"4-11 b: xnbbbbnbbbb",
+"14-16 b: bbbbbbbbbbbbbkbc",
+"4-12 w: mwrwrcpwkpzwxrj",
+"5-6 l: qlndzlcvw",
+"1-2 s: ksxk",
+"4-5 p: spgwql",
+"7-10 j: ljjjjvxjhzjjjjrj",
+"6-14 f: ffffffffffffff",
+"11-15 d: ddddddddddzdpdn",
+"3-5 t: ttmtz",
+"6-7 x: kxvxxxx",
+"3-6 m: mhmklm",
+"4-8 h: bnhjhphgqxphpnthh",
+"3-4 w: gwpq",
+"10-13 t: ttvttjhttcttrtthtb",
+"1-5 f: ffffg",
+"6-14 v: rtjjvpbvnvjwlvktvvf",
+"8-9 n: nnknnnxmnnn",
+"9-10 v: vvsvvsvvqcvvpp",
+"9-11 x: xxxxxxxxxwl",
+"3-6 h: bwqxnrhkhhwhhh",
+"4-8 l: lvllrlcf",
+"1-2 m: gtrm",
+"2-4 z: zzzz",
+"6-7 j: jjjjjjj",
+"15-16 g: sggggggggqgclgggmggw",
+"7-9 v: dvmskjsczkvjcpsv",
+"3-11 d: hmdrdfdqrddzdddjdd",
+"7-9 m: mmmwfmmmtmm",
+"6-8 x: lxsbxmlx",
+"3-5 q: qqqxq",
+"5-10 n: nnnnznnnnrn",
+"4-6 m: zmmbzg",
+"10-11 c: cpdndvlfltc",
+"2-4 k: kzkkk",
+"1-14 m: jmmmmmzmqtmsmmnmm",
+"3-13 h: gwhhrlbstldmh",
+"2-5 l: rrlhdqkzst",
+"1-6 s: sqgsjs",
+"3-4 h: hhgpb",
+"11-16 l: llllllllllllllwkllll",
+"7-11 j: gjjjtvhjjpj",
+"14-15 h: hjjvmbhfjhhxkbh",
+"1-2 t: rwxgz",
+"13-18 p: pppppppppppppppppppp",
+"1-14 x: xxrxxjxhxxxxxxbxxx",
+"4-6 b: fbzsbpbbb",
+"12-16 j: fjjgmnjwbvhjzzzs",
+"6-7 m: mmmmmtq",
+"12-16 n: nnmnnpnnnnnknqxkjnjn",
+"5-8 w: wmwwwwpwlwllzwkcw",
+"7-10 q: gbpsnqwqfm",
+"10-13 x: vxxxxxxxxxdqnxx",
+"7-12 q: qqqqqqqqqqqqqqq",
+"1-2 n: snnn",
+"6-10 l: lljllklllwp",
+"3-4 b: bsnjzbb",
+"13-14 g: tggztfgdggngmglgg",
+"6-16 l: bkcwjlwcnfwthlll",
+"8-11 s: sssssssvsss",
+"3-4 k: xmthtrcx",
+"7-8 h: wpmjhbgg",
+"4-5 p: pcpkvp",
+"14-15 s: ssrssssssssssjs",
+"14-15 x: xxxzxxgxxtxxxxqx",
+"6-7 m: nwmfmxmm",
+"6-8 k: hsknkmvhkgkkfzkjf",
+"4-5 s: sssms",
+"7-9 g: gfgqldxgxdjzglcgg",
+"10-11 k: ckkthkzpdrfv",
+"1-14 p: jptppkcppjpppppppp",
+"10-11 x: pxkccxpxdsq",
+"2-8 x: xsgxxxxvgxxk",
+"6-14 j: jjjjjzjjjjjjjtjjjj",
+"7-10 x: rxxnxrzgxxd",
+"6-12 g: dmgggpgggwczggghggm",
+"3-6 h: hdhjhhhhchh",
+"11-12 r: zrrkcrrrrrlh",
+"7-9 v: vhqvlvwvzqwqvrxvjnf",
+"1-5 r: rvmjr"
+]
+
+valid_pass = 0
+for line in (policy):
+    num,name,letters = line.split()
+    min_num,max_num = num.split("-")
+    min_num = int(min_num)
+    max_num = int(max_num)
+    value = name[0]
+    conv_lett = list(letters)
```

Comment:
I don't think you need to do this conversion, as strings will get treated like a list if you call list functions on them typically, but I may be wrong.

Reply:
Just saying I think you can just treat this string like a list (like you do for the `value` variable above).  Not 100% sure though.

Reply:
That was failing.  it may have been something else I was doing wrong OR something in the count method that didn't like that.  I thought python treats strings natively as lists that can be sliced by placement.  At least, I remember learning that and thought it was a smart feature